cmd/root.go

```diff
@@ -85,13 +85,13 @@ func getOcClient() *occlient.Client {
 // exit code of 1.
 // If the context is provided, then that is printed, if not, then the cause is
 // detected using errors.Cause(err)
-func checkError(err error, context string) {
+func checkError(err error, context string, a ...interface{}) {
```

Comment:
why are we adding `a ...interface{}` in the end? The user can format the string and pass it, right?

Reply:
Yes, for formatting so you can use formatting string with `checkError`  and you don't have to use `fmt.Sprintf` everytime you need a parameter